src/ursa/cli/__init__.py

```diff
@@ -127,5 +138,185 @@ def version() -> None:
     print(get_version("ursa-ai"))
 
 
+# TODO: add help
+@app.command()
```

Comment:
```suggestion
@app.command(help="Start MCP server to serve ursa agents")
```

@mikegros Can you edit the help here as needed?